### DISCUSSION TITLE
Possible extra reference?

### DISCUSSION THREAD

**Discussion by samchouse** [vercel/turborepo#493] [Help] [answered]
Been looking over TurboRepo source and I was wondering if this is an extra reference since all the other unmarshals don't have that extra `&`?

https://github.com/vercel/turborepo/blob/c0300f87bfd5db91d00499ad7e1b77c0aae3bd55/cli/internal/client/client.go#L200

**Comment by jaredpalmer** [comment, marked as answer]
This isn't event used anymore (it's legacy from the old turborepo.com pre-acquisition when we used the device code auth grant). Can be safely deleteed.

**Reply by samchouse** [reply]
Ah ok, so in other instances where I find unused functions can I assume it's usually legacy code?
